README.md

```diff
@@ -22,15 +22,18 @@
 Windows is not an explicitly supported environment, although where possible Palisade has been made compatible.  
 For Windows developer environments, we recommend setting up [WSL](https://docs.microsoft.com/en-us/windows/wsl/).
 
+## Getting started
+For an overview of Palisade, start with root documentation [Palisade README](https://github.com/gchq/Palisade#readme).
+
 # Palisade Clients
 
 ## Client Implementations
+Implementation of a client will need to send requests into Palisade and process the response.  
+The workflow for this is expected to follow the pattern of first sending in a request for data.  
+This will return a response consisting of a token (a unique id) that can then be used a second request to retrieve the processed data.
```

Comment:
```suggestion
This will return a response consisting of a token (a unique id) that can then be used in a second request to retrieve the processed data.
```